cibuildwheel/macos.py

```diff
@@ -203,7 +206,7 @@ def build(project_dir, output_dir, test_command, test_requires, test_extras, bef
             ])
             # Fix some weird issue with the shebang of installed scripts
             # See https://github.com/theacodes/nox/issues/44 and https://github.com/pypa/virtualenv/issues/620
-            virtualenv_env.pop('__PYVENV_LAUNCHER__', None)
+            # virtualenv_env.pop('__PYVENV_LAUNCHER__', None)  # No need for this anymore, as '__PYVENV_LAUNCHER__' is already removed from `env` above
```

Comment:
Any reason to keep this here, but commented? Otherwise, I'd say lets remove it and move the comment up to line 138

Reply:
Mainly because this issue hit us before, but then manifesting itself in the context of nested virtualenvs (I think), so I thought it was good to keep this comment at this location. Feel free to rewrite or move it, if you prefer, though!

Reply:
I see. I've made the change with an adapted comment